### DISCUSSION TITLE
Copilot Not Connecting to Server

### DISCUSSION THREAD

**Discussion by ajm2004** [community/community#51544] [Copilot Conversations]
### Select Topic Area

Bug

### Body

_GitHub Copilot could not connect to server. Extension activation failed: "At this time, Copilot is not available in your location"_

I have been using Copilot for the past few days without any issues and today out of nowhere it gave me the above msg on trying to enable it. What is the possible solution to get over this other than using VPN?
Btw I live in UAE .

**Comment by grinderek** [comment]
I got this error today with and without vpn, no idea what happened

**Reply by ajm2004** [reply]
Oh… then I guess it should be a global server issue

**Reply by AmrAbuelhamd** [reply]
I have the same issue too.

**Reply by unspectd** [reply]
I have the same issue. Stopped working today

**Reply by fraybabak** [reply]
I have same issue in turkey, Istanbul: 
```
[INFO] [auth] [2023-03-31T10:56:49.176Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-03-31T10:56:49.177Z] GitHub Copilot could not connect to server. Extension activation failed: "At this time, Copilot is not available in your location"
```

**Reply by ktKongTong** [reply]
same issue 😢

**Reply by rergards** [reply]
Same in Serbia

**Reply by zhangle9012** [reply]
The same issue in China for student account

**Reply by jose1500s** [reply]
same in mexico student account.

**Reply by fraybabak** [reply]
where is the support team?

**Reply by SocketSomeone** [reply]
Same

**Reply by MrOplus** [reply]
same in Oman

**Reply by QAQQL** [reply]
Can you tell me your area?
I am in China.

**Reply by LeeeeT** [reply]
I had the same problem until today. Now Copilot is working fine. Can you check if it works for you too?

**Reply by QAQQL** [reply]
已经确认了,是账户信息不全引发的封禁,请完善下面网址的Payment information 栏,补全您的First name和Last name
https://github.com/settings/billing/payment_information

We have confirmed that the ban was caused by incomplete account information. Please improve the Payment information section of the following website and complete your First and Last names
https://github.com/settings/billing/payment_information

**Reply by unspectd** [reply]
Can confirm, it works now
